soc/arm/silabs_exx32/Kconfig

```diff
@@ -27,7 +27,32 @@ config SOC_PART_NUMBER
 	  that you should not set directly. The part number selection choice defines
 	  the default value for this string.
 
-config HAS_EMU
+choice SOC_GECKO_DCDC_MODE
```

Comment:
We need to move this choice option below
```
config SOC_GECKO_EMU_DCDC
```
one. Otherwise the configuration interface available via menuconfig would be quite confusing. Also it should be renamed to 'SOC_GECKO_EMU_DCDC_MODE' since we decided to keep the 'EMU_'.

Reply:
Let's change the name of this option to 'SOC_GECKO_EMU_DCDC_MODE', the 'EMU_' is missing.